src/components/Navbar/Navbar.styles.css

```diff
@@ -0,0 +1,61 @@
+.navbar {
+  height: 6rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
```

Comment:
This styling approach  can create conflicts with the names, if you define `.navbar` class in another file the one class will be overridden, this is one reason why people use `styled-components` or the `material-ui` approach